front-end/src/app/shared/models/transaction-types/LOAN_RECEIVED_FROM_INDIVIDUAL_RECEIPT.model.ts

```diff
@@ -17,13 +19,9 @@ export class LOAN_RECEIVED_FROM_INDIVIDUAL_RECEIPT extends SchATransactionType {
   schema = schema;
   override useParentContact = true;
   override inheritedFields = [
+    ...INDIVIDUAL_FIELDS,
+    ...ORG_FIELDS,
     'entity_type',
```

Comment:
I think you're missing purpose description.  that would mean that ...CORE_FIELDS would cover the rest of the string literals

Reply:
But if I add the CORE_FIELDS, wouldn't I get the 'purpose_description', which I don't want?